ui/src/views/People/Add.vue

```diff
@@ -30,71 +30,19 @@
 <script>
 import axios from "axios";
 import DynamicForm from "@/components/Form/DynamicForm.vue";
+import MobileLayout from "@/mixins/MobileLayout.js";
 
 export default {
-computed:{
-    fontSizeParagraph(){
-      var fontSize='';
-      switch (this.$vuetify.breakpoint.name) {
-        case 'xs': fontSize = '12px'
-          break;
-        case 'sm': fontSize =  '12px'
-          break;
-        case 'md': fontSize =  '14px'
-          break;
-        case 'lg': fontSize =  '16px'
-          break;
-        case 'xl': fontSize =  '16px'
-          break;
-      }
-      return fontSize;
+  mixins: [MobileLayout],
+  computed:{
+    setFontSizeParagraph(){
+      return this.fontSizeParagraph(this.$vuetify.breakpoint.name);
     },
-    fontSizeH1(){
-      var fontSize = '32px';
-      switch (this.$vuetify.breakpoint.name) {
-        case 'xs': fontSize = '26px';
-          break;
-        case 'sm': fontSize = '26px'
-          break;
-        case 'md': fontSize = '32px'
-          break;
-        case 'lg': fontSize = '32px'
-          break;
-        case 'xl': fontSize = '32px'
-          break;
-      }
-      return fontSize;
+    setGridLayout(){
```

Comment:
Same comment as earlier. You aren't setting anything here.

Reply:
Done